### ISSUE TITLE
Getting help message whatever I enter

### ISSUE THREAD

**Issue by vzabauski** [pendrica/azure-credentials#3]
I installed it with `gem install azure-credentials` and after entering `azure-credentials` or a command from example I'm getting help message. Seems like a bug.

```
[######@puppet bin]$ azure-credentials -u ############# -p "############" -o ~/azure.conf -t puppet -v
-l: info is not included in the list ['debug', 'info', 'warn', 'error', 'fatal']
Usage: /home/vagrant/bin/azure-credentials (options)
    -l, --log_level LEVEL            Set the log level (debug, info, warn, error, fatal) (included in ['debug', 'info', 'warn', 'error', 'fatal'])
    -v, --verbose                    Display the credentials in STDOUT after creation? (warning: will contain secrets)
    -o, --output FILENAME            Enter the filename to save the credentials to
    -p, --password PASSWORD          Enter the password for the Azure AD user
    -r, --role ROLENAME              Enter the built-in Azure role to add the service principal to on your subscription (default: Contributor) (included in ['Contributor', 'Owner'])
    -s, --subscription ID            Enter the Subscription ID to work against (default: process all subscriptions within the Azure tenant)
    -t, --type OUTPUTTYPE            Set the output type (default: chef) (included in ['chef', 'puppet', 'terraform', 'generic'])
    -u, --username USERNAME          Enter the username (must be an Azure AD user)
    -h, --help                       Show this message
```

**Comment by stuartpreston** [comment]
I agree. It looks like some behaviour has changed in mixlib-cli or we're setting it up incorrectly.

**Comment by vzabauski** [comment]
I fixed it by installing the exact version of gems from dependencies json = 1.8.2, mixlib-cli = 1.5.0

**Comment by vzabauski** [comment]
I fixed it by installing the exact version of dependencies json 1.8.2, mixlib-cli 1.5.0.

**Comment by stuartpreston** [comment]
This issue is fixed in the latest version 0.1.2 - thanks for the feedback.
